tests/app/v2/broadcast/test_post_broadcast.py

```diff
@@ -103,12 +103,14 @@ def test_valid_post_cap_xml_broadcast_returns_201(
     assert response_json['personalisation'] is None
     assert response_json['service_id'] == str(sample_broadcast_service.id)
 
-    # TEMPORARY: while we repurpose "areas"
+    assert len(response_json['areas']['names']) == 1
```

Comment:
Really minor, but this assertion could be removed since the assertion below (`assert response_json['areas']['names'] == ['River Steeping in Wainfleet All Saints']`) shows there is only one.

Reply:
No problem: I've removed it from the third commit.